src/components/Navigation/Navigation.styled.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+import theme from "../../theme";
+
+export const SideNav = styled.nav`
+  background: white;
+  border: 2px solid ${theme.colors.gray200};
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  max-height: 500px;
```

Comment:
I don't think it's the best approach to limit this to a 500px max-height, take a look at the LeftNav code, it fills the remaining space

Reply:
I took a look at the `LeftNav` component and its height its set to take the 100%, so I set a knob to play with the height of the parent component of the `Navigation`